zwo_fixer/internal.hpp

```diff
@@ -242,10 +263,12 @@ static const OffsetMap_t g_Offsets_v0_06_0414;
 static const OffsetMap_t g_Offsets_v0_06_0328;
 
 static const VersionMap_t g_KnownLibASIVersions = {
+	{ "1, 18",       &g_Offsets_v1_18      }, // 2021-04-23
```

Comment:
I have to manually check this shit every time, because you never know when they'll pull one of these.

SDK 1.16? Version string is "1, 16, 0".
SDK 1.17? Version string is "1, 17". (Same deal with 1.18.)

**WHY** 😡